src/app/view/components/login-as/login-as.scss

```diff
@@ -1,14 +1,12 @@
 @import "mixins";
 @import "variables";
 
-:host p-dialog {
-    overflow: visible;
+form {
+    width: 300px;
```

Comment:
The `dot-dialog` have no min-width or anything it will get the width from it contents so, you can set the width in the content component or you can use the `contentStyle` attr to pass css to the div of the content in the dot-dialog